enterprise/server/api/api_server.go

```diff
@@ -345,6 +345,7 @@ func (s *APIServer) GetLog(ctx context.Context, req *apipb.GetLogRequest) (*apip
 
 	resp, err := eventlog.GetEventLogChunk(ctx, s.env, chunkReq)
 	if err != nil {
+		log.Errorf("Encountered error getting event log chunk: %s\nRequest: %s", err, chunkReq.String())
```

Comment:
p sure you don't need to call .String() explicitly here -- it will be called for you to format your thing for %s. (Same below!)